packages/default-storage-expo-plugin/.eslintrc.js

```diff
@@ -0,0 +1,2 @@
+// @generated by expo-module-scripts
+module.exports = require('expo-module-scripts/eslintrc.base.js');
```

Comment:
are there any special lint rules in this plugin? If not, let's use one we have in this repo